tests/test-extensions/vegan-extension/src/main/java/org/mule/test/vegan/extension/VeganFidelityOperation.java

```diff
@@ -19,4 +22,8 @@ public String tryEat(@XmlHints(allowReferences = false) Object food) {
     throw new IllegalArgumentException("I SHALL NEVER EAT " + food.toString());
   }
 
+  public String tryToEatThisListOfMaps(List<Map<String, String>> food) {
```

Comment:
does it affect the schema if we have a pojo that is only used by this map generic? I don't think it would, but if it does is a bug in the schema builder and we should update it too in order to avoid inconsistent schema generation.